main.tex

```diff
@@ -832,44 +869,37 @@
 
 \begin{enumerate}[label=(\alph*)]
 
-\item  drive in a safe and legal manner that observers all signs that are on Common Property in relation to the use of any vehicle;
+\item  drive or ride in a safe and legal manner that observers all signs that are on Common Property in relation to the use of any vehicle;
 
-\item  park wholly within their allocated car parking space;
+\item  park wholly within their allocated vehicle parking space(s);
 
 \item  not park in a Lot that is not designated to them without the written permission of the relevant Occupier, a copy of which must be provided to the Manager on request;
 
 \item  not at any time exceed 5 kilometers per hour when on Common Property or any slower speed notified by the Owners Corporation from time to time;
 
-\item  not drive in any manner that that is reckless, negligent or dangerous;
-
-\item  not use or permit to be used any part of the car space allocated for use other than to park a motor vehicle;
+\item  not drive or ride in any manner that that is reckless, negligent or dangerous;
 
-\item  only use their car parking space for the parking of registered, insured and roadworthy vehicles;
+\item  not use or permit to be used any part of the parking space allocated for use other than to park a vehicle or road-worthy object;
 
-\item  not lease or licence the car space without the written consent of the Owners Corporation which may be subject to the lease or licence of the car space being granted to only another Lot Owner or Occupier and the lease/licence must terminate on or before the lessee/licensee vacating the Lots;
+\item  only use their vehicle parking space for the parking of roadworthy vehicles;
 
-(i) not sub-let any car parking space to a person who does not reside within the Development;
+\item  not lease or licence the parking space without the written consent of the Owners Corporation which may be subject to the lease or licence of the parking space being granted to only another Lot Owner or Occupier and the lease/licence must terminate on or before the lessee/licensee vacating the Lots;
 
-\item  not leave any unregistered or uninured vehicle on a Lot or on the Common Property;
+(i) not sub-let any parking space to a person who does not reside within the Development;
 
-\item  not load or unload vehicles other than entirely within the Development at the locations and at times which cause minimum interference to other vehicles and other than strictly within any rules of use;
 \newpage
 
 
 
-\item  ensure that its car space remains free from oil, coolant, grease, contaminants or other materials or substances;
-
-\item  remove any vehicle from its car parking space that is not used for any period exceeding 4 weeks or which otherwise becomes dangerous or unsightly;
```

Comment:
What is wrong with leaving a vehicle in a spot for 4 weeks? Who is the victim?

What vehicles are "unsightly" is entirely subjective, and there is already an old-school combi van that stays in the garage, so this rule about unsightly vehicles is not actually being enforced.